examples/media_picker_bot.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from pyAitu import executor, Bot, Dispatcher
+from pyAitu.models import Message, Options, Form, Header, FormClosed, ValidationRule, Submit, FormAction, MediaPicker
+
+API_TOKEN = 'YOUR_API_TOKEN'
+
+bot = Bot(token=API_TOKEN)
+dp = Dispatcher(bot)
+
+logging.basicConfig(level=logging.INFO)
+
+
+@dp.message_handler()
+async def send_ui(message: Message):
+    header = Header(
+        _type="toolbar",
+        title="Title",
+        options=Options(
+            closeable=True
+        )
+    )
+    image_picker = MediaPicker(
+        content_id="image_picker1",
+        title="Media image picker title",
+        options=Options(
+            media_type="photo",
```

Comment:
```suggestion
            media_type=Media.Photo,
```
может типа такого alias сделаете чтоб ограничить ботописателя?

Reply:
done